lib/ecto_enum.ex

```diff
@@ -57,18 +64,34 @@ defmodule EctoEnum do
     end
   end
 
+  defmodule UndeterminableStorageError do
```

Comment:
Hmm. That begs the question: is the error specific to the individual member, or to the entire definition? The way I wrote the explantation would indicate the former. That said, the latter may be more useful, i.e. indicating all members that are of the wrong type.